src/plugins/recordTypes/person/forms/default.jsx

```diff
@@ -0,0 +1,218 @@
+            </Panel>
+          </Field>
+        </Field>
+        <Field name="raceGroupList">
+          <Field name="raceGroup">
+            <Panel>
+              <Row>
+                <Field name="declinedToAnswerRace" />
+                <Field name="suppliedRaces">
+                  <Field name="suppliedRace" />
+                </Field>
+                <Field name="useRestrictionRace" />
+              </Row>
+            </Panel>
+          </Field>
+        </Field>
+        <Field name="ethnicityGroupList">
+          <Field name="ethnicityGroup">
+            <Panel>
+              <Row>
+                <Field name="declinedToAnswerEthnicity" />
+                <Field name="suppliedEthnicities">
+                  <Field name="suppliedEthnicity" />
+                </Field>
+                <Field name="useRestrictionEthnicity" />
+              </Row>
+            </Panel>
+          </Field>
+        </Field>
+        <Field name="sexualityGroupList">
+          <Field name="sexualityGroup">
+            <Panel>
+              <Row>
+                <Field name="declinedToAnswerSexuality" />
+                <Field name="suppliedSexualities">
+                  <Field name="suppliedSexuality" />
+                </Field>
+                <Field name="useRestrictionSexuality" />
+              </Row>
+            </Panel>
+          </Field>
+        </Field>
+        <Field name="birthPlaceGroupList">
+          <Field name="birthPlaceGroup">
+            <Panel>
+              <Row>
+                <Field name="declinedToAnswerBirthPlace" />
+                <Field name="suppliedBirthPlace" />
+                <Field name="useRestrictionBirthPlace" />
+              </Row>
+            </Panel>
+          </Field>
+        </Field>
+        <Field name="suppliedBirthDateGroupList">
+          <Field name="suppliedBirthDateGroup">
+            <Panel>
+              <Row>
+                <Field name="declinedToAnswerBirthDate" />
+                <Field name="suppliedStructuredBirthDateGroup" />
+                <Field name="useRestrictionBirthDate" />
+              </Row>
+            </Panel>
+          </Field>
+        </Field>
+        <Field name="otherGroupList">
+          <Field name="otherGroup">
+            <Panel>
+              <Row>
+                <Field name="informationAuthor" />
+                <Field name="informationDate" />
+                <Field name="informationUseRestriction" />
+              </Row>
+              <Field name="otherInformation" />
+            </Panel>
+          </Field>
+        </Field>
+      </Panel>
+
+      <Panel name="hierarchy" collapsible collapsed>
+        <Field name="relation-list-item" subpath="rel:relations-common-list" />
+      </Panel>
+
+    </Field>
+  );
+};
+
+export default (configContext) => ({
+  messages: defineMessages({
+    name: {
+      id: 'form.person.default.name',
+      defaultMessage: 'Standard Template',
+    },
+  }),
```

Comment:
We're not intending to override `messages`, so it's best to remove it. Same comment applies to place, and the PRs that do this in other plugins.

Reply:
Yep, makes sense. I overlooked that when grabbing the forms from core. I just pushed commits removing the `messages` override.